File: restconf.go
package lang

import (
	"context"

	"github.com/freeconf/lang/pb"
	"github.com/freeconf/restconf"
	"github.com/freeconf/restconf/device"
)

type RestconfService struct {
	pb.UnimplementedRestconfServer
	d *Driver
}

func (s *RestconfService) NewServer(ctx context.Context, req *pb.NewServerRequest) (*pb.NewServerResponse, error) {
	d := s.d.handles.Require(req.DeviceHnd).(*device.Local)
	server := restconf.NewServer(d)
	resp := pb.NewServerResponse{
		ServerHnd: s.d.handles.Put(server),
	}
	return &resp, nil
}

File: python/fc/restconf.py
import fc.pb.fc_pb2
import fc.handles

class Server():

    def __init__(self, driver, device):
        self.driver = driver
        req = fc.pb.fc_pb2.NewServerRequest(deviceHnd=device.hnd)
        resp = self.driver.g_restconf.NewServer(req)
        self.hnd = driver.obj_weak.store_hnd(resp.serverHnd, self)
